CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Changed
 
 - The SDK now uses logr as the default logger to unify the logging output with the controller-runtime logs. Users can still use a logger of their own choice. See the [logging doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/logging.md) on how the SDK initializes and uses logr.
+- Add `image` flag to `test local` subcommand ([#768](https://github.com/operator-framework/operator-sdk/pull/768))
```

Comment:
Need to update the CLI reference flags for `operator-sdk test local` to include the `--image` flag and hyperlink to that here.
https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#local